main/src/ca/uwaterloo/flix/language/ast/KindedAst.scala

```diff
@@ -298,6 +301,8 @@ object KindedAst {
 
   case class Case(sym: Symbol.CaseSym, tpe: Type, sc: Scheme, loc: SourceLocation)
 
+  case class StructField(sym: Symbol.StructFieldSym, tpe: Type, sc: Scheme, loc: SourceLocation)
```

Comment:
Fields should not have type schemes, but that will probably become clear later :)

Reply:
(A struct can be polymorphic, but a field itself cannot be: its type will be determined by the type of the struct).